packages/table/src/headers/_headers.scss

```diff
@@ -47,6 +49,12 @@
       rgba($header-selected-background-color, 0.1)
     );
   }
+
+  &.pt-loading {
+    @include disable-interaction();
+    display: flex;
+    flex-direction: column;
```

Comment:
see here's `vertical-center()` minus one line so you can't use the mixin 😢 
may as well not have the mixin at all.